file path,line #,comment,satd
nginx-proxy/test/conftest.py,25,ugly global state to consider containers' IPv6 address instead of IPv4,SATD
nginx-proxy/test/conftest.py,31,,not
nginx-proxy/test/conftest.py,32,,not
nginx-proxy/test/conftest.py,33,utilities,not
nginx-proxy/test/conftest.py,34,,not
nginx-proxy/test/conftest.py,35,,not
nginx-proxy/test/conftest.py,144,"not default bridge network, fallback on first network defined",not
nginx-proxy/test/conftest.py,157,"not default bridge network, fallback on first network defined",not
nginx-proxy/test/conftest.py,222,custom DNS resolvers,not
nginx-proxy/test/conftest.py,229,"call on original DNS resolver, with eventually the original host changed to the wanted IP address",not
nginx-proxy/test/conftest.py,246,"pytest is running within a Docker container, so we do not want to remove that particular container",not
nginx-proxy/test/conftest.py,339,figure out our container networks,not
nginx-proxy/test/conftest.py,342,make sure our container is connected to the nginx-proxy's network,not
nginx-proxy/test/conftest.py,362,figure out our container networks,not
nginx-proxy/test/conftest.py,365,disconnect our container from the given network,not
nginx-proxy/test/conftest.py,380,find the list of docker networks,not
nginx-proxy/test/conftest.py,385,,not
nginx-proxy/test/conftest.py,386,,not
nginx-proxy/test/conftest.py,387,Py.test fixtures,not
nginx-proxy/test/conftest.py,388,,not
nginx-proxy/test/conftest.py,389,,not
nginx-proxy/test/conftest.py,407,give time to containers to be ready,not
nginx-proxy/test/conftest.py,432,,not
nginx-proxy/test/conftest.py,433,,not
nginx-proxy/test/conftest.py,434,Py.test hooks,not
nginx-proxy/test/conftest.py,435,,not
nginx-proxy/test/conftest.py,436,,not
nginx-proxy/test/conftest.py,438,pytest hook to display additionnal stuff in test report,not
nginx-proxy/test/conftest.py,448,"Py.test `incremental` marker, see http://stackoverflow.com/a/12579625/107049",not
nginx-proxy/test/conftest.py,461,,not
nginx-proxy/test/conftest.py,462,,not
nginx-proxy/test/conftest.py,463,Check requirements,not
nginx-proxy/test/conftest.py,464,,not
nginx-proxy/test/conftest.py,465,,not
nginx-proxy/test/test_events.py,25,give it some time to initialize and for docker-gen to detect it,not
nginx-proxy/test/test_ssl/test_hsts.py,10,Regression test to ensure HSTS is enabled even when the upstream sends an error in response,not
nginx-proxy/test/test_ssl/test_hsts.py,11,Issue #1073 https://github.com/jwilder/nginx-proxy/pull/1073,not
nginx-proxy/test/test_ssl/test_hsts.py,28,Regression test for issue 1080,not
nginx-proxy/test/test_ssl/test_hsts.py,29,https://github.com/jwilder/nginx-proxy/issues/1080,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,7,,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,8,,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,9,Tests helpers,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,10,,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,11,,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,28,,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,29,,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,30,Tests,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,31,,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,32,,not
nginx-proxy/test/test_ssl/test_dhparam_generation.py,41,"Make sure the dhparam in use is not the default, pre-generated one",not
nginx-proxy/test/test_ssl/test_dhparam.py,11,,not
nginx-proxy/test/test_ssl/test_dhparam.py,12,,not
nginx-proxy/test/test_ssl/test_dhparam.py,13,Tests helpers,not
nginx-proxy/test/test_ssl/test_dhparam.py,14,,not
nginx-proxy/test/test_ssl/test_dhparam.py,15,,not
nginx-proxy/test/test_ssl/test_dhparam.py,61,,not
nginx-proxy/test/test_ssl/test_dhparam.py,62,,not
nginx-proxy/test/test_ssl/test_dhparam.py,63,Tests,not
nginx-proxy/test/test_ssl/test_dhparam.py,64,,not
nginx-proxy/test/test_ssl/test_dhparam.py,65,,not
nginx-proxy/test/test_ssl/test_dhparam.py,73,"Make sure the dhparam in use is not the default, pre-generated one",not
nginx-proxy/test/requirements/web/webserver.py,1,!/usr/bin/env python3,not
nginx-proxy/test/test_headers/test_https.py,10,Testing the handling of X-Forwarded-For,not
nginx-proxy/test/test_headers/test_https.py,23,Testing the handling of X-Forwarded-Proto,not
nginx-proxy/test/test_headers/test_https.py,36,Testing the handling of X-Forwarded-Port,not
nginx-proxy/test/test_headers/test_https.py,49,Testing the handling of X-Forwarded-Ssl,not
nginx-proxy/test/test_headers/test_https.py,62,Other headers,not
nginx-proxy/test/test_headers/test_http.py,9,Testing the handling of X-Forwarded-For,not
nginx-proxy/test/test_headers/test_http.py,22,Testing the handling of X-Forwarded-Proto,not
nginx-proxy/test/test_headers/test_http.py,35,Testing the handling of X-Forwarded-Port,not
nginx-proxy/test/test_headers/test_http.py,48,Testing the handling of X-Forwarded-Ssl,not
nginx-proxy/test/test_headers/test_http.py,61,Other headers,not
nginx-proxy/test/stress_tests/test_unreachable_network/test_unreachable_net.py,6,TODO delete this marker once #585 is merged,SATD
nginx-proxy/test/stress_tests/test_unreachable_network/test_unreachable_net.py,34,give time for the container to initialize,not
nginx-proxy/test/stress_tests/test_deleted_cert/test_restart_while_missing_cert.py,12,TODO delete this marker once those issues are fixed,SATD
nginx-proxy/test/stress_tests/test_deleted_cert/test_restart_while_missing_cert.py,30,,not
nginx-proxy/test/stress_tests/test_deleted_cert/test_restart_while_missing_cert.py,53,give time for the container to initialize,not
